### PR TITLE
Hamburger nav 2117

### LINKED ISSUE #2117: Refactor JS: Move hamburger-nav html script to the js folder
### Overview
As a member of the website development team, Javascript code is much more maintainable and reusable if they exist in a separate folder rather than embedded with html.

### Action Items

- [x] Review the notes below to understand recent changes to our architecture
- [x] Move the script code [here](https://github.com/hackforla/website/blob/gh-pages/_includes/hamburger-nav.html) to a JS file in the [main JS directory](https://github.com/hackforla/website/tree/gh-pages/assets/js) with an appropriate name.
- [x] Call the newly made JS file in the original html file in place of the moved script. 
- [x] Ensure that the website and the hambuger nav functions the same way. No functional or visual changes should happen.
  - [x] If this is not the case, then edit the code as appropriate to maintain functionality.
### Resources/Instructions
[How do I include liquid/global variables defined using Jekyll in a Javascript file?](https://stackoverflow.com/questions/58517425/how-do-i-include-liquid-global-variables-defined-using-jekyll-in-a-javascript-fi)

#### Architecture Notes

The idea behind the refactor is to get all of our website's JS code into an organized, consolidated state. Currently, we want our JS files to exist in our [assets/js ](https://github.com/hackforla/website/tree/gh-pages/assets/js)directory with these specifications:

- In the main js directory, the js files should ideally have names corresponding to a related .html page (for example, updates.js should connect to updates.html).
- In the elements sub-directory, the js fjiles should correspond to code needed for our [design system elements ](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)to function.
- In the utility sub-directory, the js files represent reusable code that can be used in multiple files. If any js code is reused, they belong in utility, and called in a separate file in the main directory.

That said, this structure is not set in stone. If any part of it feels strange, or you have questions, feel free to bring it up with the team so we can evolve this format!

### PR DESCRIPTION
**Pull request by GLRJr**
Fixes #2117

### What changes did you make and why did you make them ?

  -Moved script in Hamburger nav HTML file and moved it to it own separate javascript file in assets folder
  -
  -

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image

</details>

### CODE REVIEW

**Review by macho-catt — CHANGES_REQUESTED** [review]
For the most part, this is good! Good job. Just one change that I recommend, the title of the front matter for hamburger-nav.js should be more descriptive than just "my js file." Could you change it to something like "Hamburger Nav?"
Additionally, your PR post has broken image links.
Thanks

File: assets/js/hamburger-nav.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: my js file
```
Comment (macho-catt) [inline assets/js/hamburger-nav.js]: Change title to be more descriptive

Comment (Aveline-art) [reply assets/js/hamburger-nav.js]: I think the title can actually be removed since it is not used in the code, as far as I can tell?

**Comment by GLRJr** [comment]
Sorry I made the changes earlier in the week but didn't actually push it.

**Review by macho-catt — APPROVED** [review]
Looks good. Great job!

**Pull request merged**